common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/azureactivedirectory/AzureActiveDirectoryOAuth2Configuration.java

```diff
@@ -42,5 +42,19 @@ public void setAuthorityHostValidationEnabled(boolean authorityHostValidationEna
 
     private boolean mAuthorityHostValidationEnabled = true;
 
-
+    /**
+     * @return The authority if configured, empty string otherwise.
+     */
+    public String getAuthority() {
+        return mAuthority;
+    }
+    
+    /**
+     * Sets an authority to use as a base for URIs as appropriate.
+     */
+    public void setAuthority(String authority) {
+        mAuthority = authority;
+    }
+    
+    private String mAuthority = "";
```

Comment:
This change looks awfully similar to work @shoatman has done in this PR: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/218/files#diff-9e0be2be55bd29586c7ece6853cbb8d6

I think we should reconcile which of these implementations we go with, since there's clearly some value here